### DISCUSSION TITLE
YouTube Music / Spotify provider - paste URL to play on MA

### DISCUSSION THREAD

**Discussion by rwjack** [music-assistant/support#3120] [Q&A] [answered]
I was trying to find a specific mix on YT Music, and even though I typed the exact media name in MA search, it couldn't be found.

There should be a way to play media by pasting a direct link in MA, be it a YouTube link, or a Spotify song link for example.

To further extend this, some sort of context menu addon for browsers would be perfect. Watching YouTube on my PC, and having a right click context menu option - "Play in MA", which opens MA (URL configurable within the browser addon itself), and this addon would then use the initially mentioned feature to paste in and play media from the desired URL.

**Comment by OzGav** [comment, marked as answer]
You can already do this https://music-assistant.io/ui/#global-search

**Reply by rwjack** [reply]
Ahh, that wasn't as obvious, but good thinking on implementing it!

**Reply by rwjack** [reply]
Circling back on this, the search is very flaky, and when I paste in the exact YT video name, I'm not finding it anywhere.

Is it possible to just paste the URL and have MASS play that exact media? Because even then I'm not getting any relevant results.

**Reply by OzGav** [reply]
YT videos are not supported. The provider is YouTube MUSIC
